pylutron_caseta/smartbridge.py

```diff
@@ -540,22 +551,40 @@ def _handle_unsolicited(self, response: Response):
     async def _login(self):
         """Connect and login to the Smart Bridge LEAP server using SSL."""
         try:
-            await self._load_devices()
-            await self._load_buttons()
-            await self._load_lip_devices()
-            await self._load_scenes()
             await self._load_areas()
-            await self._load_occupancy_groups()
-            await self._subscribe_to_occupancy_groups()
-            await self._subscribe_to_button_status()
-
-            for device in self.devices.values():
-                if device.get("zone") is not None:
-                    _LOG.debug("Requesting zone information from %s", device)
-                    response = await self._request(
-                        "ReadRequest", f"/zone/{device['zone']}/status"
-                    )
-                    self._handle_one_zone_status(response)
+
+            # Read /project to determine bridge type
+            project_json = await self._request("ReadRequest", f"/project")
+            project = project_json.Body["Project"]
+
+            if (project["ProductType"] == "Lutron Smart Bridge Project"):
```

Comment:
For QSX, this string is "Lutron HWQS Project".

Reply:
I'll add this to my fork, and we can test it out.
Are you using the Hassio OS?

Reply:
Open an issue for QSX Compatability on my fork, and once we get it working we'll ask johninaustin to merge it.  We can discuss the QSX integration there.

https://github.com/danaues/pylutron-caseta/tree/ra3

Reply:
Created https://github.com/danaues/pylutron-caseta/issues/1 with some initial notes about what I've tried and what help I could use.